scripts/speech_recognition/convert_hf_dataset_to_nemo.py

```diff
@@ -255,11 +256,11 @@ def convert_offline_dataset_to_nemo(
             if 'file' in sample:
                 del sample['file']
 
-            manifest_f.write(f"{json.dumps(sample)}\n")
+            manifest_f.write(f"{json.dumps(sample, ensure_ascii=cfg.ensure_ascii)}\n")
```

Comment:
If you are using ensure ASCII, note that you cannot have the trailing \n that this code will generate. Manifest will not be parsed correctly if there is trailing \n + ensure ascii

Reply:
Could you explain how so @titu1994? We've been disabling it when formatting the esperanto dataset as done here: https://github.com/andrusenkoau/NeMo/blob/esperanto_example/docs/source/asr/examples/esperanto_asr/scripts/get_commonvoice_data_v2.py#L96

Do you mean when `ensure_ascii` is True? This is the default value afaik (so we've been using it for all our manifests).

Reply:
You can simulate it with a manifest built with ensure ASCII false and a trailing new line, then run the char, and use ASR manifest processor to parse it. It will crash inside of the collection

Reply:
Done! thanks @titu1994 :)